src/plugins/discord/config.test.js

```diff
@@ -25,6 +25,8 @@ describe("plugins/discord/config", () => {
         },
       },
       includeNsfwChannels: true,
+      defaultReactionWeight: 0,
+      applyAveragingToReactions: true,
     };
     const parsed: DiscordConfig = parser.parseOrThrow(raw);
     expect(parsed).toEqual(_upgrade(raw));
```

Comment:
Do we need to add expect blocks to check that the value passed matches the value parsed?

Reply:
`expect(parsed).toEqual(_upgrade(raw));` kinda tests it but to be fair, it isn't a very strong assertion. I'll make that better.

Reply:
I missed the `expect(parsed).toEqual(_upgrade(raw));` part, but since `_upgrade` has its own test and it's not broken then I don't think it's necessary to add another assertion, I might however be concerned with the dependency on `_upgrade`, I dunno let me know what you think